s64da_benchmark_toolkit/correctness.py

```diff
@@ -41,8 +41,8 @@ def to_html(self):
             return status
 
         # HTML here, since it'll be used for reporting to HTML
-        truth_html = self.truth.to_html()
-        result_html = self.result.to_html()
+        truth_html = 'None' if type(self.truth) == type([]) and self.truth == [] else self.truth.to_html()
+        result_html = 'None' if type(self.result) == type([]) and self.result == [] else self.result.to_html()
```

Comment:
Please introduce a function, can be inside this function to make it more readable. Also, the type checking should IMO be:

    isinstance(self.truth, list)